app/components/explore/side-bar.js

```diff
@@ -172,6 +182,8 @@ export default class SideBar extends Component {
       cfs               : null,
       event_name        : null,
       is_online         : null,
+      has_logo    : 'false',
+      has_image   : 'false',
```

Comment:
Why dtrinI believe this has wrong indentation? Please check, I'm reviewing on Mobile

Reply:
ohh sorry. i give replace all command. thats why identation got wrong. sorry